packages/auth-ui/src/components/LoginForm.tsx

```diff
@@ -64,6 +67,7 @@ export function LoginForm({
             variant="outline" 
             className="w-full flex items-center justify-center gap-3"
             type="button"
+            onClick={openGoogleLogin}
           >
             <GoogleIcon className="w-5 h-5" />
             <span className="font-medium">Sign in with Google</span>
```

Comment:
Should we add this to the SignupForm as well? I think usually social logins are offered as an option in both login & signup.